autoblocks/testing/models.py

```diff
@@ -1,3 +1,4 @@
+from autoblocks._impl.testing.models import BaseEvaluator  # noqa: F401
```

Comment:
@adamnolte i will tackle "exporting" these from an `evaluators` module as a separate task since we'll need to be careful to leave the old exports for backwards compatability